pkg/apis/servicecatalog/types.go

```diff
@@ -60,8 +60,29 @@ type ServiceBrokerSpec struct {
 	// CABundle is a PEM encoded CA bundle which will be used to validate a Broker's serving certificate.
 	// +optional
 	CABundle []byte
+
+	// RelistBehavior specifies the type of relist behavior the catalog should exhibit
```

Comment:
We need to add validations for these fields.  See pkg/apis/servicecatalog/validation/broker.go